src/module.ts

```diff
@@ -68,6 +68,10 @@ export default defineNuxtModule<SanityModuleOptions>({
       options.useCdn = process.env.NODE_ENV === 'production' && !options.token
     }
 
+    if (options.perspective === 'previewDrafts' && !options.useCdn) {
+        logger.info("To use perspective: previewDrafts, useCdn must be set to true")
```

Comment:
Could we just set the value and print a message letting the user know?

Reply:
don't you think there might also be a perspective set per client, e.g. if the user has a custom preview client as well as a main one?

Reply:
Yes I see that now, check my implementation in client.ts

Reply:
Done

Reply:
thank you ❤️

nice